packages/venia-ui/lib/components/CheckoutPage/ShippingInformation/EditForm/editForm.gql.js

```diff
@@ -1,6 +1,7 @@
 import gql from 'graphql-tag';
 
 import { ShippingInformationFragment } from '../shippingInformationFragments.gql';
+import { ShippingMethodsFragment } from '@magento/venia-ui/lib/components/CheckoutPage/ShippingMethod/shippingMethodFragments.gql';
```

Comment:
Whoops, fix import to relative.

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/2280/commits/4622f28ad800516b494c3fa5ff7a8bba6f01a9ee.